pkg/provider/pulumi/azure/schedule.go

```diff
@@ -0,0 +1,105 @@
+package azure
+
+import (
+	"encoding/json"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/nitrictech/cli/pkg/cron"
+	"github.com/nitrictech/cli/pkg/provider/pulumi/common"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/logic"
+	"github.com/pulumi/pulumi-azure/sdk/v4/go/azure/eventgrid"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type SchedulesArgs struct {
+	ResourceGroupName pulumi.StringInput
+	Topics            map[string]*eventgrid.Topic
+}
+
+type Schedules struct {
+	pulumi.ResourceState
+
+	Name      string
+	WorkFlows map[string]*logic.Workflow
+}
+
+// https://docs.microsoft.com/en-us/azure/logic-apps/concepts-schedule-automated-recurring-tasks-workflows
+// https://docs.microsoft.com/en-us/azure/connectors/connectors-create-api-servicebus
+func (a *azureProvider) schedules(ctx *pulumi.Context, name string, args *SchedulesArgs, opts ...pulumi.ResourceOption) (*Schedules, error) {
+	res := &Schedules{
+		Name:      name,
+		WorkFlows: map[string]*logic.Workflow{},
+	}
+	err := ctx.RegisterComponentResource("nitric:schedules:AzureWorkflows", name, res, opts...)
+	if err != nil {
+		return nil, err
+	}
+
+	ia, err := logic.NewIntegrationAccount(ctx, name+"ia", &logic.IntegrationAccountArgs{
+		ResourceGroupName: args.ResourceGroupName,
+		Tags:              common.Tags(ctx, name+"ia"),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	for _, s := range a.s.Schedules {
+		if len(args.Topics) == 0 || s.Target.Type != "topic" || s.Target.Name == "" {
+			continue
+		}
+		trig, err := cron.ConvertToAzure(s.Expression)
+		if err != nil {
+			return nil, err
+		}
+
+		trigObj := map[string]interface{}{}
+		err = mapstructure.Decode(trig, trigObj)
+		if err != nil {
+			return nil, err
+		}
+		/*
+			err := a.schedule(ctx, k, s.Expression, topics[s.Target.Name])
+			if err != nil {
+				return errors.WithMessage(err, "schedule "+k)
+			}
+		*/
+		worflowJSON, err := json.Marshal(map[string]interface{}{
+			"$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+			"version": "2.0",
+			"extensionBundle": map[string]interface{}{
+				"id":      "Microsoft.Azure.Functions.ExtensionBundle.Workflows",
+				"version": "[1.*, 2.0.0)",
+			},
+			"extensions": map[string]interface{}{
+				"serviceBus": map[string]interface{}{
+					"batchOptions": map[string]interface{}{
+						"operationTimeout": "00:15:00",
+					},
+				},
+			},
+			"triggers": map[string]interface{}{
+				"Recurrence": map[string]interface{}{
+					"recurrence": trigObj,
+					"type":       "Recurrence",
+				},
+			},
```

Comment:
just need to figure out the queue send "action"
https://docs.microsoft.com/en-us/azure/connectors/connectors-create-api-servicebus#add-service-bus-action
I might need to use the UI then see if I can export the json to see the format